scripts/h5toobj.py

```diff
@@ -186,21 +131,11 @@ def generate_chunks(shape, dtype, target_object_size, dims_to_split=(0, 1)):
     return tuple(chunks)
 
 
-def dask_from_chunks(chunks, out_name, base_name):
-    single_chunk = all(len(c) == 1 for c in chunks)
-    last_index_per_dim = [np.r_[0, np.cumsum(c)[:-1]][-1] for c in chunks]
-    widths = [len(str(i)) for i in last_index_per_dim]
+def dsk_from_chunks(chunks, out_name):
     keys = list(product([out_name], *[range(len(bds)) for bds in chunks]))
```

Comment:
What exactly is `chunks` in this case?

Reply:
Chunks is a dask concept: a tuple of tuple of ints that describes the splits along each dimension, e.g.

```Python
((1, 1, 1, 1), (512, 512, 512, 512), (480,), (2,))
```

It is basically an exploded shape specifier. All of this will be better documented once these functions settle in `VisFlagsWeights` and other places - `h5toobj` is just a fetid breeding ground.